xls2xml/amp-t2d/T2D_xlsx.schema

```diff
@@ -0,0 +1,97 @@
+#Schema file for defining validation rules for each field
+#
+#The validation rules could be found at:
+#http://docs.python-cerberus.org/en/stable/validation-rules.html
+
+Project:
+  schema:
+    PI email(s):
+      type: string
+      nullable: False
+      required: True
+      regex: '^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+(;[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)*$'
+    Submitter Email(s):
+      type: string
+      nullable: False
+      required: True
+      regex: '^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+(;[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+)*$'
+    Project Publications:
+      type: string
+      nullable: True
+      regex: '^PubMed:[0-9]+(;PubMed:[0-9]+)*$'
+    Total number of files being submitted:
+      type: number
+      nullable: False
+      min: 1
+
+Cohort:
+  schema:
+    External Links:
+      type: string
+      nullable: True
+      regex: '[a-zA-Z0-9]+:[a-zA-Z0-9]+'
+    "# T2D Cases":
+      type: number
+      min: 0
+    "# T2D Controls":
+      type: number
+      min: 0
+
+Phenotype_Trait:
+  schema:
+    Age:
+      type: float
+      nullable: False
+      min: 0.1
+
+Sample:
+  schema:
+    Sample_ID:
+      type: string
+      regex: '^SAM\d+$'
+      nullable: False
+    Subject_ID:
+      type: string
+      regex: '^SAM\d+$'
+      nullable: True
```

Comment:
Subject ID is required, but the can default to sample ID is not given.

Reply:
Yes, "nullable: True" means that the value could be empty.